src/libaktualizr/uptane/secondaryconfig.h

```diff
@@ -37,16 +37,18 @@ class SecondaryConfig {
   std::string ecu_public_key;
   KeyType key_type{KeyType::kRSA2048};
 
+  // FIXME: secondary config contains secondary specific params
+  // so introduction of any new type of secondary will require changes here/libaktualizr
+  // what we would like to avoid
```

Comment:
Yep, this is not ideal. It might be a necessary evil for now, but we should try to remove them if possible. Not the top priority, though.

Reply:
OK